model/src/node_id.rs

```diff
@@ -21,6 +21,13 @@ impl ParseError {
     }
 }
 
+#[derive(Clone, Debug, thiserror::Error, PartialEq, Serialize, Deserialize)]
+#[error("NodeId `{input:?}` parsing error: {msg}")]
```

Comment:
This will be ugly printed :(

Reply:
Vec<u8> does not support `:x` printing :(